eth/api.go

```diff
@@ -1501,13 +1503,14 @@ func (api *PublicDebugAPI) SeedHash(number uint64) (string, error) {
 // PrivateDebugAPI is the collection of Etheruem APIs exposed over the private
 // debugging endpoint.
 type PrivateDebugAPI struct {
-	eth *Ethereum
+	config *core.ChainConfig
+	eth    *Ethereum
 }
 
 // NewPrivateDebugAPI creates a new API definition for the private debug methods
 // of the Ethereum service.
-func NewPrivateDebugAPI(eth *Ethereum) *PrivateDebugAPI {
-	return &PrivateDebugAPI{eth: eth}
+func NewPrivateDebugAPI(config *core.ChainConfig, eth *Ethereum) *PrivateDebugAPI {
+	return &PrivateDebugAPI{config: config, eth: eth}
```

Comment:
It will cause too much noise changing this proper. We've decided to leave as is so that we can properly fix up the chain maker and together with that change fix up the additional configuration fields.

Changing this temporarily would involve creating a `core.BlockChain` for every call of `GenerateChain`, which involves setting up verifiers, processors, proof of work, a Genesis block, etc.